.circleci/test-targets.txt

```diff
@@ -4,7 +4,7 @@
 //stratum/glue/status/...
 //stratum/hal/bin/bcm/standalone/...
 //stratum/hal/config/...
-//stratum/hal/lib/barefoot:bf_chassis_manager_test
+//stratum/hal/lib/barefoot/...
```

Comment:
Refer to this commit: 013ea707911403fcb173738265af27a763e33449

Reply:
I see that this creates an issue